### PR TITLE
Collect jobs and cronjobs

### PR DESCRIPTION
**Pull request by xlucas**
### What does this PR do?

Add job and cronjob collection to the orchestrator check.

Those collectors are not enabled in the default list.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -229,3 +229,5 @@ replace (
 )
 
 replace github.com/vishvananda/netlink => github.com/DataDog/netlink v1.0.1-0.20210312173533-6d628a7fc6f3
+
+replace github.com/DataDog/agent-payload => github.com/DataDog/agent-payload v4.65.1-0.20210415142421-a378ec3176fa+incompatible
```
Comment (xlucas) [inline go.mod:233]: Note: dropping this and pinning to the right version when https://github.com/DataDog/agent-payload/pull/95 gets merged.

File: go.mod [hunk 2]
```diff
@@ -229,3 +229,5 @@ replace (
 )
 
 replace github.com/vishvananda/netlink => github.com/DataDog/netlink v1.0.1-0.20210312173533-6d628a7fc6f3
+
+replace github.com/DataDog/agent-payload => github.com/DataDog/agent-payload v4.65.1-0.20210415170107-6d2b6dc55680+incompatible
```
Comment (xlucas) [inline go.mod:233]: Note: dropping this and pinning to the right version when https://github.com/DataDog/agent-payload/pull/95 gets merged.

File: pkg/collector/corechecks/cluster/orchestrator/orchestrator_processor.go [hunk 3]
```diff
@@ -24,12 +24,87 @@ import (
...
+			GroupId:     groupID,
+			GroupSize:   int32(groupSize),
+			ClusterId:   clusterID,
+			Tags:        cfg.ExtraTags,
+		})
+	}
+
+	log.Debugf("Collected & enriched %d out of %d cron jobs in %s", len(cronJobMsgs), len(cronJobList), time.Since(start))
+	return messages, nil
+
```
Comment (mfpierre) [inline pkg/collector/corechecks/cluster/orchestrator/orchestrator_processor.go:86]: nit:extra line

File: pkg/collector/corechecks/cluster/orchestrator/orchestrator_transformer.go [hunk 4]
```diff
@@ -54,6 +56,80 @@ func extractDeployment(d *v1.Deployment) *model.Deployment {
 	return &deploy
 }
 
+func extractJob(j *batchv1.Job) *model.Job {
```
Comment (mfpierre) [inline pkg/collector/corechecks/cluster/orchestrator/orchestrator_transformer.go:59]: this function could use a unit test

File: pkg/collector/corechecks/cluster/orchestrator/orchestrator_transformer.go [hunk 5]
```diff
@@ -54,6 +56,80 @@ func extractDeployment(d *v1.Deployment) *model.Deployment {
...
+		job.Status.StartTime = j.Status.StartTime.Unix()
+	}
+	if j.Status.CompletionTime != nil {
+		job.Status.CompletionTime = j.Status.CompletionTime.Unix()
+	}
+
+	return &job
+}
+
+func extractCronJob(cj *batchv1beta1.CronJob) *model.CronJob {
```
Comment (mfpierre) [inline pkg/collector/corechecks/cluster/orchestrator/orchestrator_transformer.go:103]: ditto for the unit test

**Review by mfpierre — APPROVED** [review]
Logic LGTM 👍

**Pull request merged**